src/modules/comment/commentCounter.test.js

```diff
@@ -0,0 +1,40 @@
+const { JSDOM } = require('jsdom');
+const { default: commentCounter } = require('./commentCounter.js');
+
+const dom = new JSDOM();
+global.document = dom.window.document;
+// const commentCounter = require('./commentCounter.js');
```

Comment:
- [ ] Same here, Please remove uncommented code to make your code more readable and clean.